common/build.gradle

```diff
@@ -1,12 +1,27 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.google.gradle:osdetector-gradle-plugin:1.6.0'
+    }
+}
+
 plugins {
     id 'java'
     id 'maven'
     id 'com.google.protobuf' version '0.8.5'
```

Comment:
Add the following line here, and delete the `buildscript` block above:

    id 'com.google.osdetector' version '1.6.0'

(Per https://github.com/google/osdetector-gradle-plugin#for-gradle-21-and-higher)

Reply:
Ok, that's a shame. Would be worth a comment about this in the `buildscript` block; otherwise it just looks like it got added due to ignorance of how `plugins {}` works.

Reply:
And you should be able to use `jcenter()` in the buildscript block vs `mavenCentral()`.